### PR TITLE
docs: update readme and interested parties

### PR DESCRIPTION
**Pull request by camigira**
## 📑 Description
👋 
I'm Camilo Giraldo and would like to get involved with the project.
- Added myself to interested parties.
- Removed **_Interested Parties_** section in Readme as it seemed to be a duplicate of **Getting started** and a broken link to Notion.
- Added link /k8sgpt/CONTRIBUTING.md into **_Contributing_** section.

## ✅ Checks

- [ ] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,16 +8,12 @@ If you are interested to be informed about the project or to contribute, feel fr
...
 ## Contributing
 
-We welcome contributions from the community! Please refer to our Contributing Guidelines for more information on how to get involved.
+We welcome contributions from the community! Please refer to our [Contributing Guidelines](../k8sgpt/CONTRIBUTING.md) for more information on how to get involved.
 
 ## Code of Conduct
 
 We have a [Code of Conduct](./CODE_OF_CONDUCT.md) that all contributors are expected to follow. Please read it before contributing.
 
-## Interested Parties
```
Comment (AlexsJones) [inline README.md:17]: Why is this removed?

Comment (camigira) [reply README.md:17]: This text seems redundant with the first  **Getting Started** section and has a broken link (https://www.notion.so/467a8f2a4d8e4e7ab5f7c7bc6460b1c9).

> ## Getting Started
> If you are interested to be informed about the project or to contribute, feel free to add yourself and/or your organization to [Interested Parties](./interested-parties.md).

**Review by AlexsJones — APPROVED** [review]

**Review by AlexsJones — APPROVED** [review]
